### DISCUSSION TITLE
Bind a reset button to a function that changes all the buttons from DISABLED to NORMAL and sets button text as " ".

### DISCUSSION THREAD

**Discussion by alamir3ali** [TomSchimansky/CustomTkinter#633] [Q&A]
Hello, I am doing a tic-tac-toe project with 9 cells so 81 buttons in total. The structure of code is following:

- Class Board, 1 object 

- Class Cell, 9 objects; 9 micro cells are associated for each object  

- Class Micro_cell, 81 objects

Firstly I was using tkinter and defined a clear_element() function for the reset button:
 
`#reset functions: 

def clear_element(element):
   for widgets in element.winfo_children():
      widgets.destroy()`

However, after switching the code from tkinter to customtkinter the reset button stopped working. 

To illustrate, for example we have this situation in the game and we want to restart: 

![image](https://user-images.githubusercontent.com/114143531/202578361-2f0538b1-79f3-4536-8370-00ac2d3c3869.png)

and after clicking reset button we have:

![image](https://user-images.githubusercontent.com/114143531/202580212-b504b58d-5320-4b31-b978-c7f556e5bf7f.png)

Which is kind of sad. 

If you need more information, just tell me.

All help would be appreciated and thank you.

**Comment by Akascape** [comment]
@alamir3ali 
Do not destroy the widget, configure them instead.
```python
def clear_element(element):
    for widgets in element.winfo_children():
        widgets.configure(state="normal", text=" ")
```

**Reply by alamir3ali** [reply]
Thank you, it works, however it has this animation that is unpleasant to my eyes:
![image](https://user-images.githubusercontent.com/114143531/202743274-40fb00b7-9a24-4e93-b0c7-5ace6bda6899.png)

**Reply by Akascape** [reply]
@alamir3ali Can you post the full code?
